src/kp2bw/bw_serve.py

```
@@ -0,0 +1,578 @@
+        when the ``bw serve`` JSON envelope reports ``success: false``.
+        """
+        resp = self._http.request(
+            method,
+            path,
+            json=json_body,
+            params=params,
+        )
+        if resp.status_code >= 400:
+            raise BitwardenClientError(
+                f"bw serve returned HTTP {resp.status_code} for {method} {path}"
+            )
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)